### PR TITLE
Upgrade MBT Maven Builder Default Command to Batch Mode

### PR DESCRIPTION
**Pull request by young-yang03**
## Description

As a build tool, MBT wrap all modules’s build process internal, all builders, such as golang, maven and nodejs, run build command in batch mode is resonable.

In this PR:

(1) we update maven builder default command from current “mvn clean package” to “mvn -B clean package”.

(2) fix the binwrap install problem is included in this PR from @wangshen-sap 

(3) upgrade sap-jvm version from 8.1.094 to 8.1.095

### Checklist
- [ ] Code compiles correctly
- [ ] Relevant tests were added (unit / contract / integration)
- [ ] Relevant logs were added
- [ ] Formatting and linting run locally successfully
- [ ] All tests pass
- [ ] UA review
- [ ] Design is documented
- [ ] Extended the README / documentation, if necessary
- [ ] Open source is approved

### CODE REVIEW

**Review by wangshen-sap — APPROVED** [review]

File: docs/docs/whatsnew.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
 # <b>Important updates</b>
 
+## v1.2.25
+
+### Configuration of `maven` builder has changed. 
+As of version 1.2.25, the `mvn -B clean package` command is used where the `maven` builder is configured for building a module or in a global build step. 
+
+The `-B` parameter is added to `maven` builder's build command which will start maven in a interactive mode.
```
Comment (ShiraManas) [inline docs/docs/whatsnew.md:8]: The `-B` parameter is added to the `maven` builder's build command, which will start maven in interactive mode.

(Does the build command start maven in interactive mode? or the -B parameter? Just trying to understand how to say this most clearly.)

File: docs/docs/whatsnew.md [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+
+As a build tool, MBT packages all modules’s build process internal, all native builders, such as golang, maven and nodejs, run build command in batch mode is resonable.
```
Comment (ShiraManas) [inline docs/docs/whatsnew.md:10]: Can you explain this? Not sure I understood.
As a build tool, MBT packages all modules’s build process internal, all native builders, such as golang, maven and nodejs, run build command in batch mode is resonable.

File: docs/docs/whatsnew.md [hunk 3]
```diff
@@ -1,5 +1,30 @@
...
+
+<b>NOTE:</b>  The `maven` builder is configured implicitly for the `java` module type.
+
+If you want to keep the previous behavior, that is, to apply the `mvn clean package` command, you need to change the build parameters of the relevant module in one of the following ways:
```
Comment (ShiraManas) [inline docs/docs/whatsnew.md:14]: I only see one way described. Will more be added? If not, I'd say this:
If you want to keep the previous behavior, that is, to apply the `mvn clean package` command, you need to change the build parameters of the relevant module by configuring the `custom` builder:
(Remove line 16 and continue with Line 17 -name...)

**Review by ShiraManas — COMMENTED** [review]
Hi. I've added some suggestions and questions. Thanks!

File: docs/docs/whatsnew.md (hunk 1 above)
Comment (young-yang03) [reply docs/docs/whatsnew.md:8]: By adding -B parameter, the "maven clean package" command will start in interactive mode

File: docs/docs/whatsnew.md (hunk 2 above)
Comment (young-yang03) [reply docs/docs/whatsnew.md:10]: MBT builds MTA (Multiple Target Application), MTA contains many kind of modules, each module will be build by native builders, such as golang for go type module, maven for java type module, nodejs for nodejs type module.

MBT is an build tool, when MTA application is build, all build process for different module is packaged into MBT, it wrapped all build process internal and it should not run in an interactive modle, it is reasonable for MBT exec the build process in a batch modle.

File: docs/docs/whatsnew.md (hunk 3 above)
Comment (young-yang03) [reply docs/docs/whatsnew.md:14]: yes, you are right,thanks

File: docs/docs/whatsnew.md (hunk 1 above)
Comment (ShiraManas) [reply docs/docs/whatsnew.md:8]: Ok. It's better this way. 
By adding the -B parameter, the "maven clean package" command will start in interactive mode.

File: docs/docs/whatsnew.md (hunk 2 above)
Comment (ShiraManas) [reply docs/docs/whatsnew.md:10]: Ok. Can we use this?

MBT builds the MTA (Multitarget Application), which contains many type of modules. Each module is built by native builders, such as Golang for the Go module, Maven for the Java module, and Node.js for the Node.js module.

When the MTA is built, all build processes for different modules are packaged into the MBT. It wraps all the build processes as internal and it should not run in interactive mode. It is reasonable for MBT to execute the build process in batch mode.

**Review by ShiraManas — COMMENTED** [review]
Looks good. I added some last suggestions.

File: docs/docs/whatsnew.md (hunk 1 above)
Comment (young-yang03) [reply docs/docs/whatsnew.md:8]: ok, Done

File: docs/docs/whatsnew.md (hunk 2 above)
Comment (young-yang03) [reply docs/docs/whatsnew.md:10]: OK, I do a little simplification based on your advise

As a build tool, MBT builds the MTA (Multi-Target Application), which contains many type of modules. Each module is built by corresponding builder, such as maven, nodejs and go.

When the MTA is built, all build processes for different modules are packaged into the MBT. It wraps all the build processes as internal and it should not run in interactive mode. It is reasonable for MBT to execute the build process in batch mode.

**Comment by young-yang03** [comment]
@ShiraManas based on your advise, I update the whatsnew.md, pls help to confirm it, thanks!

File: docs/docs/whatsnew.md [hunk 4]
```diff
@@ -1,5 +1,30 @@
 # <b>Important updates</b>
 
+## v1.2.25
+
+### Configuration of `maven` builder has changed. 
+As of version 1.2.25, the `mvn -B clean package` command is used where the `maven` builder is configured for building a module or in a global build step. 
+
+By adding the -B parameter, the "maven clean package" command will start in interactive mode.
+
+As a build tool, MBT builds the MTA (Multi-Target Application), which contains many type of modules. Each module is built by corresponding builder, such as maven, nodejs and go.
```
Comment (ShiraManas) [inline docs/docs/whatsnew.md:10]: Multitarget Application 
(we write it without the hyphen)

Comment (ShiraManas) [inline docs/docs/whatsnew.md:10]: which contains many types of modules.

Comment (ShiraManas) [inline docs/docs/whatsnew.md:10]: such as maven, node.js, and go.
(Doesn't it need to have the period in node.js? Also, when I looked up these builders, they were all listed with capital first letter- Maven, Node.js, and Go. Is that required here?)

**Review by ShiraManas — COMMENTED** [review]
Looks great! I added just a few last comments.

File: docs/docs/whatsnew.md (hunk 4 above)
Comment (young-yang03) [reply docs/docs/whatsnew.md:10]: I think maven, npm and golang is better for native builder name.

As to the builder name, you can reference : https://github.com/SAP/cloud-mta-build-tool/blob/master/configs/builder_type_cfg.yaml

In the above doc, all builder's name is low case， maven is for java type module, golang for go type module, npm for nodejs type module

**Pull request merged**